components/workspacekit/cmd/rings.go

```diff
@@ -113,6 +120,7 @@ var ring0Cmd = &cobra.Command{
 		cmd.Env = append(os.Environ(),
 			"WORKSPACEKIT_FSSHIFT="+prep.FsShift.String(),
 			fmt.Sprintf("WORKSPACEKIT_NO_WORKSPACE_MOUNT=%v", prep.FullWorkspaceBackup || prep.PersistentVolumeClaim),
+			"GITPOD_WORKSPACE_ID="+wsid,
```

Comment:
We don't have to set this envvar, because we're passing `os.Environ()`.
This way we're setting it twice (same below)

Reply:
@csweichel Good point! Thank you. I fixed that.